protostar/argument_parser/unparser.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Optional
+
+
+def unparse_arguments_from_external_source(value: Any) -> Optional[list[str]]:
+    """Arguments from external sources need to be unparsed in order to be parsed by custom parsers."""
+    if value is None:
+        return None
+    if isinstance(value, bool):
```

Comment:
What's the purpose of leaving out the boolean values? I don't understand the case.

Reply:
"Bool" represents flags. Flags are treated a bit differently because they don't accept values.
| TOML | CLI |
| --- | --- |
| `arg-name = 42` | "42" |
| `flag-name = True` | ? |

I should've used assert with a description here.

Reply:
Done. Added support for parsing and unparsing for booleans despite argparse not accepting explicit values for flags.